### PR TITLE
feat: Ipfs 404 page

### PR DESCRIPTION
**Pull request by svvimming**
Ipfs 404 page added to the `/static` directory.

### CODE REVIEW

**Review by 2color — APPROVED** [review]
LGTM 

e.g. https://bafybeif4jrceyplxzylsc46jwk6qluku6ky2ku37tuxfwn3t4ttkwnmcfu.on.fleek.co/project/brave-sdfdsg

**Review by lidel — CHANGES_REQUESTED** [review]
nit: 

- avoid adding adding fonts for the second time (these files are already in `assets/fonts/Inter` and referenced in [main.scss](https://github.com/ipfs/ecosystem-directory/blob/82ac048af95ae53c57134eed69764fa0cd653d6f/assets/scss/main.scss#L8), please deduplicate) 
-  since #238  landed,  links to `https://ipfs.io`  and `https://docs.ipfs.io` should be replaced with `https://ipfs.tech`  and `https://docs.ipfs.tech`

File: assets/scss/main.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // /////////////////////////////////////////////////////////////////////// Fonts
 // -----------------------------------------------------------------------------
-$fontAssetPath: '~assets/fonts';
+$fontAssetPath: '~static/fonts';
```
Comment (orvn) [inline assets/scss/main.scss:3]: Nice, this works great. Now the font files will always live in one place, unaffected by the static site compilation.

**Review by orvn — APPROVED** [review]
Tested, lgtm!

**Pull request merged**